i18n/pt.yaml

```diff
@@ -0,0 +1,78 @@
+404_title: '404'
+404_description: 'Ops! Esta página não existe!'
+subheader: ''
+
+# menu ----
+home: "Home"
+about-us: 'About'
+mission: "Mission"
+get-involved: 'Get involved'
+global-team: 'Meet the global team'
```

Comment:
another comment but in portuguese, it's on purpose? 😅

Reply:
same on purpose as above :)